packages/ui/src/components/Button/button.style.ts

```diff
@@ -58,11 +55,31 @@ export const ButtonStyles = styled.div`
     line-height: 22px;
     padding: 0px 10px;
```

Comment:
I see I have missed these before. Sorry. Lets make these padding. for xm -> 8px for sm -> 12px for md -> 24px and for lg -> 32px

Reply:
It is done, this is also related to the padding being multiples of 4. I will also read about it. Thanks